.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+<!--
+# openms/openms-docs pull request
+
+Thank you for thinking about contributing to OpenMS Documentation!
+
+Please fill in the appropriate checklist below.
+
+Remember that PRs should be made against the staging branch, unless you're preparing a hotfix to be released.
+
+Learn more about contributing to OpenMS Documentation: https://staging-openms.readthedocs.io/en/staging/.github/CONTRIBUTING.html
+-->
+
+## Describe the change
+
+<!-- Please add a brief description about the change in documentation that you're suggesting -->
+
+## PR checklist
+
+- [ ] I have added description of the change I'm proposing in the OpenMS Documentation.
+- [ ] I have read and followed [OpenMS Documentation Contributing guidelines](CONTRIBUTING.md).
+- [ ] I have attached a screenshot of the relevant area after this change.
+- [ ] `CHANGELOG.md` is updated.
```

Comment:
This is definitely a nitpick but this repository doesn't have a `CHANGELOG.md` file, it only has a `CHANGELOG` file. Most people would understand what you mean but it may confuse a small percentage of people who can't find `CHANGELOG.md` and maybe they'll be nervous to change the `CHANGELOG` file.

My suggestion is to either rename the current changelog file to include the `.md` prefix or remove the `.md` from this bullet point.

(Sorry if someone already said this, there were a lot of comments to read through)

Reply:
please feel free to add comments or create tickets, even if they are repetitive.